src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -261,8 +263,8 @@ function useChatUtilsHook(): UseChatUtilsHook {
     }));
   };
 
-  const goToContactScreen = () => {
-    navigation.navigate('ContactScreen');
+  const goToContactScreen = ({from}: ContactScreenPayload) => {
+    navigation.navigate('ContactScreen', {from});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `goToContactScreen` now accepts an object with a `from` property as a parameter. This change allows the source of navigation to be passed to the 'ContactScreen', which can be useful for handling different scenarios based on where the navigation originated. However, ensure that all calls to this function have been updated to pass this new parameter.
```diff
- const goToContactScreen = () => {
-   navigation.navigate('ContactScreen');
+ const goToContactScreen = ({from}: ContactScreenPayload) => {
+   navigation.navigate('ContactScreen', {from});
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->